dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,597 @@
+from __future__ import division
+
+import logging
+import math
+from warnings import warn
+
+import numpy as np
+from sklearn.metrics.scorer import check_scoring
+from sklearn.utils import check_random_state
+from tornado import gen
+
+from ._incremental import IncrementalSearchCV
+from ._successive_halving import SuccessiveHalvingSearchCV
+
+logger = logging.getLogger(__name__)
+
+
+def _get_hyperband_params(R, eta=3):
+    """
+    Parameters
+    ----------
+    R : int
+        The maximum number of iterations desired.
+
+    Returns
+    -------
+    N : list
```

Comment:
Te returns here aren't correct. IIUC, just a `Dict[int, Tuple[list, list]]` is returned